README.md

```diff
@@ -191,7 +191,8 @@ In general:
   }
   ```
 
-- For method declarations, use 4 space indentation for its parameters when they don't fit in a single line. Return types can be either on the same line as the last parameter, or put to next line with 2 space indent.
+- For method declarations, use 4 space indentation for its parameters and put each in each line when they don't fit in two lines. Return types can be either on the same line as the last parameter, or put to next line with 2 space indent.
```

Comment:
"indentation for its parameters " => "indentation for their parameters" (an existing bug)

"when they" => "when the parameters"

"or put to next line with 2 space indent" => "or start a new line with 2 space indent."